### ISSUE TITLE
Various "too many open files" and other errors

### ISSUE THREAD

**Issue by zramsay** [blocklayerhq/chainkit#9]
After a successful `chainkit create demoapp`, the next step leads to the following:

![screen shot 2019-01-08 at 15 25 57](https://user-images.githubusercontent.com/8304391/50856867-d3ec3300-1359-11e9-9ee3-3f4066acf98d.png)

two attempts at `chainkit start` lead to different "too many open files" errors. Not also the "dial backoff" error. Restarted computer and was able to get past these errors but then:

![screen shot 2019-01-08 at 15 31 06](https://user-images.githubusercontent.com/8304391/50857070-7e645600-135a-11e9-882f-0d9627e49d71.png)

Perhaps the "mdns lookup error" can be safely ignored, if so then maybe it should be a warning? Then the third variation of a "too many files" error.

Note: I have the minimum apps/files/etc open after the restart, and the defaults according to https://superuser.com/questions/433746/is-there-a-fix-for-the-too-many-open-files-in-system-error-on-os-x-10-7-1  ... to what should (if at all) increase them? 

Going to try running on a fresh DO box.

**Comment by samalba** [comment]
Yes, this is coming from ipfs so we can't alter the logs. And it should be flagged as a warning since this is optional from chainkit's pov.

I'll flag this as enhancement since it would improve the readability of the logs.
